js/interface.js

```diff
@@ -3059,8 +3059,20 @@ $('#fl-store-certificate').on('change', function () {
 // Firebase
 
 $('#fl-store-firebase').on('change', function () {
-  appStoreFirebaseFileField = this;
   var fileName = this.value.replace(/\\/g, '/').replace(/.*\//, '');
+  var fileExtension = (/[.]/.exec(fileName)) ? /[^.]+$/.exec(fileName)[0] : undefined; 
```

Comment:
why not using `.match` so you only need one check to also get the extension?

Reply:
@squallstar Fixed